modules/virt-configuring-default-cpu-model.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-default-cpu-model.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-default-cpu-model_{context}"]
+= Configuring the default CPU model
+
+Configure the `defaultCPUModel` by updating the `HyperConverged` custom resource (CR). You can change the `defaultCPUModel` while OpenShift Virtualization is running.
+
+Note: The `defaultCPUModel` is case sensitive.
+
+.Prerequisites
+
+* Install the OpenShift CLI (oc).
+
+.Procedure
+
+. Open the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
```

Comment:
Can we be sure HCO is always installed in this namespace?
Maybe it's worthwhile to mention that the namespace should the namespace on which Openshift Virtualization (CNV) is installed.

But I may be over-thinking this.